recipe/meta.yaml

```diff
@@ -44,11 +88,16 @@ test:
     - pip
     - pytest
     - ipython
+    - pytorch
+    - pillow>=10.3.0
+    - setuptools <=65.5.1 # [py<312]
+    - setuptools ==68.2.2 # [py>=312]
     # To fix `from numba.np.ufunc import _internal SystemError: initialization of _internal failed without raising an exception`
     - numba >=0.57.0
   commands:
-    - pip check
-    - pytest tests -vv
+    # skipping pip check since it detects uuid6 0.0.0.dev0 when uuid6==2024.7.10 is used
```

Comment:
The package has a wrong `METADATA` file and should be patched to fix the issue.
It's caused by a specific env variable required for retrieving a version, see https://github.com/oittaa/uuid6-python/blob/2024.07.10/setup.py#L15-L19.
I think it's fine to skip it here but could you please create a Jira ticket to rebuild `uuid6`?

Reply:
ok.

Reply:
created here https://anaconda.atlassian.net/browse/PKG-6267